src/test/digest.rs

```diff
@@ -0,0 +1,38 @@
+use std::time::Duration;
+
+use super::super::*;
+
+#[test]
+fn valid_credentials() {
+    let arbitrary_username = "MyUsername";
+    let arbitrary_password = "MyPassword";
+    let digest_auth_middleware =
+        DigestAuthMiddleware::new(arbitrary_username.into(), arbitrary_password.into());
+    let test_url =
+        format!("http://httpbin.org/digest-auth/auth/{arbitrary_username}/{arbitrary_password}");
```

Comment:
Realized this only now: upstream ureq will likely want to support even slightly older Rust versions, so we should resort to the older syntax here. :smiling_face_with_tear: